MediaBrowser.Api/EnvironmentService.cs

```diff
@@ -120,13 +122,14 @@ public class EnvironmentService : BaseApiService
         /// Initializes a new instance of the <see cref="EnvironmentService" /> class.
         /// </summary>
         /// <param name="networkManager">The network manager.</param>
-        public EnvironmentService(INetworkManager networkManager, IFileSystem fileSystem)
+        public EnvironmentService(
+            ILogger<EnvironmentService> logger,
+            IServerConfigurationManager serverConfigurationManager,
+            IHttpResultFactory httpResultFactory,
+            INetworkManager networkManager,
+            IFileSystem fileSystem)
+            : base(logger, serverConfigurationManager, httpResultFactory)
         {
-            if (networkManager == null)
-            {
-                throw new ArgumentNullException(nameof(networkManager));
-            }
-
```

Comment:
why?

Reply:
Can’t be null, gets passed by the ServiceProvider